scripts/update_specs_from_pricing.py

```diff
@@ -41,7 +41,10 @@
     'South America (Sao Paulo)': 'sa-east-1',
     'US East (N. Virginia)': 'us-east-1',
     'US East (Ohio)': 'us-east-2',
+    'US East (Verizon) - Atlanta': 'us-east-1',
     'US East (Verizon) - Boston': 'us-east-1',
+    'US East (Verizon) - New York': 'us-east-1',
+    'US East (Verizon) - Washington DC': 'us-east-1',
```

Comment:
https://aws.amazon.com/about-aws/whats-new/2020/09/announcing-new-aws-wavelength-zones-in-atlanta-new-york-city-washington-dc/